python/BioSimSpace/Process/_gromacs.py

```diff
@@ -2194,6 +2194,12 @@ def _add_position_restraints(self, config):
                 # The number of restraint files.
                 num_restraint = 1
 
+                # Store the number of molecule types in the top file.
```

Comment:
Would be nice to add a test for this I think because there are quite a few operations going on in this function.

Reply:
Yes, I agree that it would be good to add a test. I'll try to come up with a small example. (Could use yours, but the files are quite large.)